app/admin/admin_users.rb

```diff
@@ -0,0 +1,28 @@
+ActiveAdmin.register AdminUser do
+  permit_params :email, :password, :password_confirmation
+
+  index do
+    selectable_column
```

Comment:
so that each row can be selected by clicking on checkbox for batch action
![Screenshot from 2022-03-01 17-14-21](https://user-images.githubusercontent.com/57142378/156163571-316dc2ef-9891-4e93-bccc-b7fd4f79356b.png)